lib/ecto/adapters/postgres.ex

```diff
@@ -41,6 +43,7 @@ defmodule Ecto.Adapters.Postgres do
     * `:port` - Server port (default: 5432)
     * `:username` - Username
     * `:password` - User password
+    * `:maintenance_database` - Default maintenance database name
```

Comment:
I would copy the description of the `createdb` command and include the default value:

> `:maintenance_database` - Specifies the name of the database to connect to when creating or dropping the  database. Defaults to `"postgres"`

What do you think?

Reply:
👍 thanks @fertapric. Fixed that.